src/appleseed/renderer/modeling/scene/scene.h

```diff
@@ -75,13 +75,13 @@ class APPLESEED_DLLSYMBOL Scene
     // Delete this instance.
     virtual void release() APPLESEED_OVERRIDE;
 
-    // Set the camera.
-    void set_camera(foundation::auto_release_ptr<Camera> camera);
-
     // Access the camera.
     // Return 0 if the camera does not exist.
     Camera* get_camera() const;
```

Comment:
Update the description of `get_camera()` to make it clear which camera is returned.

Actually, we should probably even rename all `get_camera()` to `get_active_camera()` when they return the active camera, to avoid confusion...